fabric_plugin/tests/blueprint/scripts/script.py

```diff
@@ -10,4 +10,10 @@ def test_run_python_script():
         os.environ.get('test_value')
 
 
+def test_nested_property():
+    ctx.instance.runtime_properties['test_value'] = {'dict': ['some_value']}
+    ctx.instance.runtime_properties['test_value'] = \
```

Comment:
yeah, the `_test_run_script` checks that after running the script, `props['test_value']` is equal to `'some_value'`. So we can either write the test so that the values at the end are as expected, or write a new test method (on the calling python side). So i decided it's easier to reuse the existing method, and just make the test conform to the convention it requires